clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java

```diff
@@ -99,6 +99,10 @@ public String regionFromZone(String zone) {
       return null;
     }
 
+    public List<String> getZonesFromRegion(String region) {
+      return region != null && regionToZonesMap != null ? regionToZonesMap.get(region) : null;
```

Comment:
It wasn't clear to me exactly what kind of Map regionToZonesMap will be. As
the javadocs for Map.get() indicate a Map impl may NPE on a null key, I
chose to play it safe.

On Fri, Apr 29, 2016 at 3:26 PM Travis Tomsu <notifications@github.com>
wrote:

> In
> clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java
> <https://github.com/spinnaker/clouddriver/pull/569#discussion_r61632487>:
>
> > @@ -99,6 +99,10 @@ public String regionFromZone(String zone) {
> >        return null;
> >      }
> >
> > +    public List<String> getZonesFromRegion(String region) {
> > +      return region != null && regionToZonesMap != null ? regionToZonesMap.get(region) : null;
>
> You don't need the region != null, because there is no null key in the
> map anyway.
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/clouddriver/pull/569/files/f56b5e492ef90dbc69b04922aaa838c9e8b47a12#r61632487>
>